### ISSUE TITLE
Is this GEM compatible with Workplace bots for Messenger ?

### ISSUE THREAD

**Issue by anthonyhamou** [jgorset/facebook-messenger#248]
Hi,

I am using this Ruby client code for FB messenger since months

One of my customer asked me to develop a bot on Workplace by Facebook.

Is it still possible to use this GEM ? Is is compatible with Workplace bots for Messenger ?

Here below the code error i got when i try to run my usual bot code ?

Thank you

![code error screenshot](https://user-images.githubusercontent.com/19352847/57979283-e4a16980-7a1b-11e9-9317-86297c77b109.png)

**Comment by amrutjadhav** [comment]
Hello, @anthonyhamou  I am not familiar with the Workplace API. But by seeing the error, I am can say that this bot is not compatible with Workplace. There seems to be a difference in the request parameter of normal bot and Workplace bot request.

**Comment by jgorset** [comment]
Yep, that seems about right. I've not put any effort into Workplace for this gem, because I haven't needed it, but your contributions are welcome if you do! :slightly_smiling_face:
